docs/upgrade/upgrade-enterprise/upgrade-guide-from-2022.1-to-2022.2/upgrade-guide-from-2022.1-to-2022.2-generic.rst

```diff
@@ -138,7 +138,7 @@ Download and install the new release
 
                sudo apt-get clean all
                sudo apt-get update
-               sudo apt-get dist-upgrade scylla-enterprise-server
+               sudo apt-get dist-upgrade scylla-enterprise
```

Comment:
just "install" (not dist-upgrade)
most people don't want scylla upgrade to upgrade all their packages